### PR TITLE
Issue #562 fixes

### LINKED ISSUE #562: refactor usage of deprectated `sdkerrors` package
this import `sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"`
should be refactored to `cosmossdk.io/errors`

### PR DESCRIPTION
**Pull request by muku314115**
## Description

Refactoring sdkerrors throughout the project

----

Closes #562 

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpeted math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

---

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

**Review by mtsitrin — CHANGES_REQUESTED** [review]
thanks for the contribution!
pls use consistent naming throughout the codebase
either
`errorsmod "cosmossdk.io/errors"`
or
`sdkerrors "cosmossdk.io/errors"`

**Comment by muku314115** [comment]
> thanks for the contribution! pls use consistent naming throughout the codebase either `errorsmod "cosmossdk.io/errors"` or `sdkerrors "cosmossdk.io/errors"`

thaks, I have used sdkerrors for all "cosmossdk.io/errors" and errortypes for all "github.com/cosmos/cosmos-sdk/types/errors"
is there any change required there?

**Comment by mtsitrin** [comment]
> > thanks for the contribution! pls use consistent naming throughout the codebase either `errorsmod "cosmossdk.io/errors"` or `sdkerrors "cosmossdk.io/errors"`
> 
> thanks, I have used sdkerrors for all "cosmossdk.io/errors" and errortypes for all "github.com/cosmos/cosmos-sdk/types/errors" is there any change required there?

e.g https://github.com/dymensionxyz/dymension/blob/6bdc4bc5c12f6c774a8227e07423dc6167ecd4c1/app/ante/reject_msgs.go#L4

pls rename all to `sdkerrors "cosmossdk.io/errors"`

**Comment by muku314115** [comment]
> > > thanks for the contribution! pls use consistent naming throughout the codebase either `errorsmod "cosmossdk.io/errors"` or `sdkerrors "cosmossdk.io/errors"`
> > 
> > 
> > thanks, I have used sdkerrors for all "cosmossdk.io/errors" and errortypes for all "github.com/cosmos/cosmos-sdk/types/errors" is there any change required there?
> 
> e.g
> 
> https://github.com/dymensionxyz/dymension/blob/6bdc4bc5c12f6c774a8227e07423dc6167ecd4c1/app/ante/reject_msgs.go#L4
> 
> pls rename all to `sdkerrors "cosmossdk.io/errors"`

Thanks for letting me know, added the changes

File: go.mod
Comment (mtsitrin) [inline go.mod:1]: why it modified?

File: e2e/go.mod
Comment (mtsitrin) [inline e2e/go.mod:1]: why it's modified?

**Review by mtsitrin — CHANGES_REQUESTED** [review]

File: go.mod
Comment (muku314115) [reply go.mod:1]: Goland updated some stuff after merging conflicts, reverted the go.mod files to what they were

**Pull request closed without merging**